readme.md

```diff
@@ -115,24 +115,63 @@ The example assumes we merge `mps2021.1` into `mps2021.2`.
 5. Run all tests
 6. Update build model
 
-## Publishing and Releasing
 
-Run:
+## Publishing and releasing
 
-`./gradlew release`
+Use the `publish` task to publish a _snapshot_ version (i.e., versions ending with `-SNAPSHOT`), and the `release` task to release an _official_ version.
 
-To publish to either Sonatype (for snapshot versions, i.e., versions ending with `-SNAPSHOT`) or to Maven Central.
-For doing that you need to configure your sonatype credentials in ~/.gradle/gradle.properties:
+In order to be able to publish or release, you need to do the following:
 
-```
-ossrhUsername=<username>
-ossrhPassword=<password>
-```
+1. Register on Sonatype.
+	You'll need the corresponding credentials later on, but note that these are different from the user token you'll also need (to create).
+2. Ask to be added to the list of users authorized to release under `io.lionweb`.
+  <!-- **Check**: is this still accurate? -->
+
+2. Create an _user token_ for Sonatype:
+  * Log in on [https://s01.oss.sonatype.org/](https://s01.oss.sonatype.org/) — this is **not** the same site as [Maven Central](https://central.sonatype.com/)
+  * Go to your profile (dropdown under your user name in the top-right corner)
+  * Click the “Generate User Token” (or “Access User Token” if you've already made one)
```

Comment:
I think you've missed the dropdown to switch to tokens

Reply:
Added.